### PR TITLE
[feature](merge-cloud) Add meta service proto definition

### PR DESCRIPTION
**Pull request by w41ter**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by w41ter** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

**Review by platoneko — APPROVED** [review]

**Comment by w41ter** [comment]
run buildall

File: gensrc/thrift/MasterService.thrift [hunk 1]
```diff
@@ -42,6 +42,7 @@ struct TTabletInfo {
     15: optional Types.TReplicaId replica_id
     // data size on remote storage
     16: optional Types.TSize remote_data_size
+    17: optional bool is_persistent
```
Comment (yiguolei) [inline gensrc/thrift/MasterService.thrift:45]: do not add 17, it is used.
add the field to the end

**Comment by w41ter** [comment]
run buildall

**Comment by w41ter** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**